{"name": "Получение ошибки при изменении данных для неавторизованного пользователя", "status": "passed", "description": "Тест для проверки получение ошибки при изменении данных для неавторизованного пользователя", "steps": [{"name": "Отправить запрос на обновление данных.", "status": "passed", "start": 1748676344024, "stop": 1748676344143}, {"name": "Проверить код и сообщение об ошибке", "status": "passed", "start": 1748676344143, "stop": 1748676344143}], "parameters": [{"name": "field", "value": "'name'"}], "start": 1748676344024, "stop": 1748676344144, "uuid": "0609162d-dff1-41f9-9ef0-5cd73f19352a", "historyId": "ed4dc5ecee87bf450e8ea8a60fa5b129", "testCaseId": "c305736e8c00d8dff7dc49a5cd5303ce", "fullName": "tests.test_data_change.TestUserDataUpdates#test_unauthorized_updates_error_received", "labels": [{"name": "suite", "value": "Изменение данных пользователя."}, {"name": "parentSuite", "value": "tests"}, {"name": "subSuite", "value": "TestUserDataUpdates"}, {"name": "host", "value": "DESKTOP-G7OKOON"}, {"name": "thread", "value": "832-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_data_change"}], "links": [{"type": "link", "url": "https://code.s3.yandex.net/qa-automation-engineer/python-full/diploma/api-documentation.pdf", "name": "API документация"}]}